bandit.yml

```diff
@@ -7,3 +7,7 @@ skips:
   - B101  # Ignore assert statements
   - B404  # Ignore warnings about importing subprocess
   - B603  # Ignore warnings about calling subprocess.Popen without shell=True
+  - B403  # Ignore warnings about import pickle
+  - B301  # Ignore warnings about pickle.load
+  - B303  # Ignore warnings about MD2, MD4, MD5, or SHA1 hash functions
+  - W0703 # Ignore warnings about broad-except
```

Comment:
Can we not add this to the bandit file please. I think that it is a very valid warning. We should never do `except:` or `except Exception:` as it catches everything including typos which makes debugging really hard

Reply:
yes if we can solve the problem of pickle

Reply:
If we have a case where it is necessary we can always disable the check at that line with a comment. Disabling it for the whole project is a bad idea because it stops the error being raised for this in the future

Reply:
done